Update vpc peering doc 20240706 (#18149)
pingcap/docs: tidb-cloud/set-up-vpc-peering-connections.md

    The **VPC Peering** configuration is displayed by default.

4. Click **Create VPC Peering**, choose **AWS VPC Peering**, choose the **TiDB Cloud VPC Region**, and then fill in the required information of your existing AWS VPC:

>     - Your existing VPC Region

Reviewer: ```suggestion
    - Your VPC Region
```

Reviewer: ![image](https://github.com/pingcap/docs/assets/79440533/c80d41d4-738a-43d9-8fa2-009aeef0278d)

    - AWS Account ID
    - VPC ID
    - VPC CIDR

>     You can get these information from your VPC details page on the AWS console. TiDB Cloud supports creating VPC peerings between VPCs in the same region or from two different regions.

Reviewer: ```suggestion
    You can get such information from your VPC details page on the [AWS Management Console](https://aws.amazon.com/console). TiDB Cloud supports creating VPC peerings between VPCs in the same region or from two different regions.
```

    ![VPC peering](/media/tidb-cloud/vpc-peering/vpc-peering-creating-infos.png)

> 5. Click **create**. The VPC peering request is sent and you can see the VPC Peering info in **VPC Peering** tab and the status is "System Checking".

Reviewer: ```suggestion
5. Click **Create** to send the VPC peering request, and then view the VPC peering information on the **VPC Peering** tab. The status of the newly created VPC peering is **System Checking**.
```

> 6. Check **...** of the VPC Peering record, and click **View** , you can view the connection information of your TiDB VPC peerings from **VPC Peering Details** page.

Reviewer: ```suggestion
6. To view detailed information about your newly created VPC peering, click **...** > **View**  in the **Action** column. The **VPC Peering Details** page is displayed.
```

    2. From the left navigation bar, open the **Peering Connections** page. On the **Create Peering Connection** tab, a peering connection is in the **Pending Acceptance** status.

>     3. Confirm the requester owner is TiDB Cloud (You can view the "TiDB Cloud AWS Account ID" and "TiDB Cloud VPC ID" from **VPC Peering Details** page on [TiDB Cloud console](https://tidbcloud.com)). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.

Reviewer: ```suggestion
    3. Confirm that the requester owner and the requester VPC match **TiDB Cloud AWS Account ID** and **TiDB Cloud VPC ID** on the **VPC Peering Details** page of the [TiDB Cloud console](https://tidbcloud.com). Right-click the peering connection and select **Accept Request** to accept the request in the **Accept VPC peering connection request** dialog.
```

    The **VPC Peering** configuration is displayed by default.

4. Click **Create VPC Peering**, choose **Google Cloud VPC Peering**, choose the **TiDB Cloud VPC Region**, and then fill in the required information of your existing Google Cloud VPC:

    > **Tip:**
    >
    > You can follow instructions next to the **Application Google Cloud Project ID** and **VPC Network Name** fields to find the project ID and VPC network name.

    - Google Cloud Project ID
    - VPC Network Name
    - VPC CIDR

5. Click **Create** to send the VPC peering request, and then view the VPC peering information on the **VPC Peering** tab. The status of the newly created VPC peering is **System Checking**.

> 6. To view detailed information about your newly created VPC peering, click **...** > **View**  in the **Action** column. The **VPC Peering Details** page is displayed.

Reviewer: ```suggestion
6. To view detailed information about your newly created VPC peering, click **...** > **View** in the **Action** column. The **VPC Peering Details** page is displayed.
```

    - VPC Network Name
    - VPC CIDR

> 

Reviewer 2: Change Application Google Cloud Project ID to Google Cloud Project ID
![ededa1a8-6aa8-4a19-a7f2-9aca707e5903](https://github.com/pingcap/docs/assets/35301108/6b13af5f-ddaa-4e43-8928-016248855de0)

Reviewer: Done: https://github.com/pingcap/docs/pull/18149/commits/43eb51e46463530a033a410fe1863bbb86cbca8d

> 5.  Click **create**. The VPC peering request is sent and you can see the VPC Peering info in **VPC Peering** tab and the status is "System Checking".

Reviewer: ```suggestion
5. Click **Create** to send the VPC peering request, and then view the VPC peering information on the **VPC Peering** tab. The status of the newly created VPC peering is **System Checking**.
```

> 6. Check **...** of the VPC Peering record, and click **View** , you can view the connection information of your TiDB VPC peerings from **VPC Peering Details** page.

Reviewer: ```suggestion
6. To view detailed information about your newly created VPC peering, click **...** > **View**  in the **Action** column. The **VPC Peering Details** page is displayed.
```
